homeassistant/components/blink/camera.py

```diff
@@ -56,41 +59,50 @@ def __init__(self, data, name, camera):
         _LOGGER.debug("Initialized blink camera %s", self.name)
 
     @property
-    def extra_state_attributes(self):
+    def extra_state_attributes(self) -> Mapping[str, Any] | None:
         """Return the camera attributes."""
         return self._camera.attributes
 
-    def enable_motion_detection(self) -> None:
+    async def async_enable_motion_detection(self) -> None:
         """Enable motion detection for the camera."""
-        self._camera.arm = True
-        self.data.refresh()
+        try:
+            await self._camera.async_arm(True)
+            await self.data.refresh(force=True)
```

Comment:
Same question here

Reply:
Same answer :)
Maybe a bit more elaboration, the rate limiter in the API limits messages sent, but sending an arm command is not subject to the rate limit imposed by the site.